<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>22</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveFileAs"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Úpravy</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Vzhled</string>
    </property>
    <widget class="QMenu" name="menuChangeFontSize">
     <property name="title">
      <string>Změnit velikost fontu</string>
     </property>
     <addaction name="actionChangeFontSize14"/>
     <addaction name="actionChangeFontSize16"/>
     <addaction name="actionChangeFontSize18"/>
     <addaction name="actionChangeFontSize20"/>
     <addaction name="actionChangeFontSize22"/>
     <addaction name="actionChangeFontSize24"/>
     <addaction name="actionChangeFontSize28"/>
     <addaction name="actionChangeFontSize32"/>
    </widget>
    <addaction name="menuChangeFontSize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFile">
   <property name="text">
    <string>Nový</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Uložit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="text">
    <string>Uložit jako</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Otevřít</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Zpět</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Znovu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Kopírovat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Vložit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Výjmout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionFontSize12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="ChangeFontSize_2">
   <property name="text">
    <string>14</string>
   </property>
  </action>
  <action name="actionChangeFontSize16">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="actionChangeFontSize18">
   <property name="text">
    <string>18</string>
   </property>
  </action>
  <action name="actionChangeFontSize20">
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="actionChangeFontSize22">
   <property name="text">
    <string>22</string>
   </property>
  </action>
  <action name="actionChangeFontSize24">
   <property name="text">
    <string>24</string>
   </property>
  </action>
  <action name="actionChangeFontSize28">
   <property name="text">
    <string>28</string>
   </property>
  </action>
  <action name="actionChangeFontSize32">
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="actionChangeFontSize14">
   <property name="text">
    <string>14</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
